policyengine_us/parameters/gov/states/ga/tax/income/credits/non_refundable.yaml

```
@@ -0,0 +1,14 @@
+description: Georgia provides these non-refundable income tax credits.
+values:
+  2021-01-01:
+    - ga_cdcc
+    - ga_low_income_credit
+metadata:
+  unit: list
+  period: year
+  label: Georgia non-refundable tax credits
+  reference:
+    - title: Official Code of Georgia Annotated | CH17 Income Taxes | O.C.G.A. § 48-7A-3 (b)
```

subsection c specifies nonrefundability
```suggestion
    # Low income credit.
    - title: Official Code of Georgia Annotated | CH17 Income Taxes | O.C.G.A. § 48-7A-3 (c)
```